src/jit/lowerxarch.cpp

```diff
@@ -1954,6 +1957,8 @@ void Lowering::TreeNodeInfoInitBlockStore(GenTreeBlk* blkNode)
                     // series of 16-byte loads and stores.
                     blkNode->gtLsraInfo.internalFloatCount = 1;
                     blkNode->gtLsraInfo.addInternalCandidates(l, l->internalFloatRegCandidates());
+                    // use XMM register for load and store, need set the flag for AVX instruction
```

Comment:
>// use XMM register for load and store, need set the flag for AVX instruction [](start = 20, length = 77)

I would suggest to rephrase this comment as 

"Uses XMM reg for load and store and hence check to see whether AVX instructions are used for codegen"